From a539c8edf7105dbda398465519b3216e26201f8a Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:23 -0400
Subject: [tools] perf/record: Rename --initial-delay to --delay

Message-id: <1408953210-26343-256-git-send-email-jolsa@redhat.com>
Patchwork-id: 88898
O-Subject: [PATCH RHEL7.1 BZ1133083 255/322] perf record: Rename --initial-delay to --delay
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit a6205a35ba2d56dcfb79b08cc11b8c7d549826aa
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Tue Jan 14 17:58:12 2014 -0300

description
===========
To be consistent with the equivalent option in 'stat', also, for the
same reason, use -D as the one letter alias.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/Documentation/perf-record.txt b/tools/perf/Documentation/perf-record.txt
index 3a35a85..c71b0f3 100644
--- a/tools/perf/Documentation/perf-record.txt
+++ b/tools/perf/Documentation/perf-record.txt
@@ -208,7 +208,8 @@ overrides that and uses per-thread mmaps.  A side-effect of that is that
 inheritance is automatically disabled.  --per-thread is ignored with a warning
 if combined with -a or -C options.
 
---initial-delay msecs::
+-D::
+--delay=::
 After starting the program, wait msecs before measuring. This is useful to
 filter out the startup phase of the program, which is often very different.
 
diff --git a/tools/perf/builtin-record.c b/tools/perf/builtin-record.c
index 78e790f..3c394bf 100644
--- a/tools/perf/builtin-record.c
+++ b/tools/perf/builtin-record.c
@@ -882,7 +882,7 @@ const struct option record_options[] = {
  OPT_CALLBACK('G', "cgroup", &record.evlist, "name",
        "monitor event in cgroup name only",
        parse_cgroups),
- OPT_UINTEGER(0, "initial-delay", &record.opts.initial_delay,
+ OPT_UINTEGER('D', "delay", &record.opts.initial_delay,
     "ms to wait before starting measurement after program start"),
  OPT_STRING('u', "uid", &record.opts.target.uid_str, "user",
      "user to profile"),
-- 
1.7.1